bin/vyper

```diff
@@ -76,18 +84,34 @@ elif not args.debug:
     sys.tracebacklimit = 0
 
 
-def uniq(seq):
-    exists = set()
-    return [x for x in seq if not (x in exists or exists.add(x))]
+T = TypeVar('T')
+
+
+def uniq(seq: Iterable[T]) -> List[T]:
```

Comment:
I'm aware this has nothing to do with this PR and should not be addressed here but it would probably be a good idea to not turn the `Iterable[T]` into a `List[T]` and instead do one of these things:

1. Either just yield the items so that we return an `Iterable[T]` and have consumers deal with conversion if needed
or 
2. Return a `Tuple[T, ...]` instead of a `List[T]` to not return something that is mutable.

Reply:
@cburgdorf I like the yielding suggestion, thanks!

Reply:
Great! Minor further suggestion. You could change the type annotation from `Generator[T, None, None]` to `Iterable[T]` which is basically a subset of a generator and looks a bit cleaner imho. Basically our convention in Trinity is to use `Iterator[T]` instead of `Generator[T, None, None`] whenever you are only interested in the yield type but not in the send type or return type (e.g. not using `generator.send(something)`